### DISCUSSION TITLE
Set room temperature on physical device/valve

### DISCUSSION THREAD

**Discussion by mikulcak** [jmcollin78/versatile_thermostat#251] [Q&A] [answered]
Hi,

I'm sure this has been answered before but I haven't been able to find a solution anywhere. My question is simple: When using Versatile Thermostat (configured as "over a climate") in a room with multiple radiators and therefore multiple physical radiator valves (all tuya zigbee TRVs), is it possible to control the target room temperature using physical the devices? Or must that be done via Home Assistant?

Thank you so much in advance,
Marcus

**Comment by jmcollin78** [comment, marked as answer]
Hello, yes you can act directly on TRV if you want. It is not recommended (you have possibly two contradictory orders) but it works.

**Reply by mikulcak** [reply]
Thanks for the reply - but the temperature I set on the device is not read in the backend and then sent to the other TRVs in the room, is it?

**Reply by jmcollin78** [reply]
Can you give it a try ? I think this may works out of the box.

**Reply by maia** [reply]
If you have multiple TRVs in one room and want all to update together whenever you change one physically, you could write an automation that will trigger on a change in target temperature and update the target temperature in the other devices (or: find the matching preset for that temperature and send that preset to all TRVs).
